apps/sensenet/cypress/integration/upload/upload.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Upload', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+
+  it('uploading a file/content should works properly', () => {
+    const fileToBeUploaded = 'data.json'
+    cy.get('[data-test="Content"]').click()
+    cy.get('[data-test="IT Workspace"]').click()
+    cy.get('[data-test="Document Library"]').click()
```

Comment:
```suggestion
    cy.get('[data-test="Document library"]').click({ force: true })
```
Document library has 'l' instead of 'L' (not uppercased). And xou should use force click here, cause the element has CSS pointer-events: none.